### PR TITLE
Update sbt and scala

### PR DESCRIPTION
**Pull request by cquiroz**
Mostly we get scala 3.0.0-RC3 but also the build is improved

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,14 +23,14 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        scala: [2.12.13, 2.13.4, 3.0.0-RC1, 3.0.0-RC2]
+        scala: [2.12.13, 2.13.5, 3.0.0-RC2, 3.0.0-RC3]
```
Comment (johnynek) [inline .github/workflows/ci.yml:26]: do we need RC2 still? I think we aren't changing this repo much, so I would imagine just doing RC3 would be fine.

**Comment by johnynek** [comment]
Thanks!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cquiroz) [reply .github/workflows/ci.yml:26]: Probably not, I'm just keeping this idea of releasing the last 2 versions

**Comment by cquiroz** [comment]
Would it be possible to make a release?

**Pull request merged**